crates/miden-lib/asm/kernels/transaction/lib/prologue.masm

```diff
@@ -640,30 +640,30 @@ proc.process_note_args_and_metadata
     # => [NOTE_METADATA]
 end
 
-#! Checks that the number of note inputs is within limit and stores it to memory.
+#! Checks that the note storage length is within limit and stores it to memory.
 #!
 #! Inputs:
 #!   Operand stack: [note_ptr]
-#!   Advice stack: [inputs_len]
+#!   Advice stack: [storage_len]
 #! Outputs:
 #!   Operand stack: [note_ptr]
 #!   Advice stack: []
 #!
 #! Where:
 #! - note_ptr is the memory location for the input note.
-#! - inputs_len is the note's input count.
-proc.process_note_inputs_length
-    # move the inputs length from the advice stack to the operand stack
+#! - storage_len is the length of the note's storage.
+proc.process_note_storage_length
+    # move the storage length from the advice stack to the operand stack
     adv_push.1
-    # => [inputs_len, note_ptr]
+    # => [storage_len, note_ptr]
 
     # validate the input length
     dup exec.::$kernel::util::note::get_max_inputs_per_note lte
```

Comment:
`shared_utils/util/note.masm` file should be updated as well: we store there the `MAX_INPUTS_PER_NOTE` constant and its getter, which is used here.